src/Bravo/Operation.cpp

```diff
@@ -0,0 +1,56 @@
+#include "Operation.h"
+
+#include <stdexcept>
+
+using namespace TW::Bravo;
+
+using json = nlohmann::json;
+
+/// Transfer Operation
+TransferOperation::TransferOperation(std::string from, std::string to, Asset asset, std::string memo)
+                         : from(from), to(to), asset(asset), memo(memo) {
+	validate();
+}
+
+TransferOperation::TransferOperation(std::string from, std::string to, int64_t amount, bool isTestNet, std::string memo) 
+                    : TransferOperation(from, to, Asset(amount, isTestNet), memo) { }
+
+TransferOperation::TransferOperation(std::string from, std::string to, std::string asset, std::string memo)     
+                    : TransferOperation(from, to, Asset::fromString(asset), memo) { }
+
+void TransferOperation::validate()
+{
+	if (from.size() > MaxAccountNameSize) {
```

Comment:
Indention in this file looks different, could you please try clang-format?

Reply:
Okay